cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -523,6 +526,30 @@
     },
     "system_details": {
       "properties": {
+        "container_images": {
+          "items": {
+            "properties": {
+              "description": {
+                "description": "A brief description of the container image",
+                "type": "string"
+              },
+              "recid": {
+                "description": "The internal ID of the container image, if it is another Open Data record",
```

Comment:
Let's start again by the same word as the field is called (`recid`) so that readers can form better mental association:

- Record ID of the container image (if it exists as another Open Data record)